packages/react-scripts/template/src/App.css

```diff
@@ -1,21 +1,28 @@
 .App {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  padding: 3rem;
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.App > *+* {
+  margin: 0;
+  margin-top: 1rem;
 }
 
-.App-header {
-  background-color: #222;
-  height: 150px;
-  padding: 20px;
-  color: white;
+.App-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 38vh;
 }
 
-.App-intro {
-  font-size: large;
+code {
```

Comment:
Similarly, this will behave unexpectedly when you nest components. We use class names everywhere to avoid this problem. Don't style tags.